src/openforms/authentication/contrib/demo/plugin.py

```diff
@@ -56,13 +56,17 @@ def start_login(
     def handle_return(
         self, request: HttpRequest, form: Form
     ) -> Union[str, HttpResponse]:
-        submited = self.form_class(request.POST)
-        if not submited.is_valid():
+        submitted = self.form_class(request.POST)
+        if not submitted.is_valid():
             return HttpResponseBadRequest("invalid data")
 
-        request.session[self.provides_auth] = submited.cleaned_data[self.form_field]
+        request.session["form_auth"] = {
```

Comment:
session key is defined as a constant in https://github.com/open-formulieren/open-forms/blob/master/src/openforms/authentication/constants.py#L5